packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBitcoinAddresses/selectors.js

```diff
@@ -19,20 +19,21 @@ export const getData = (state, coin) => {
     return map(formatAddress, addressesData)
   }
 
-  const getAddressesData = (coin) => {
+  const getAddressesData = (ownProps) => {
```

Comment:
This change should be on getData used in index.js, not getAddressesData used in selectors.js

Reply:
Good point, also turns out defaultProps are not passed in `ownProps` so I need to change this.